internal/indexer/postgres/affectedmanifest.go

```diff
@@ -142,13 +155,13 @@ func affectedManifests(ctx context.Context, pool *pgxpool.Pool, v claircore.Vuln
 	}
 	log.Debug().Int("count", len(filteredRecords)).Msg("vulnerable indexrecords")
 
-	// query the manifest index for manifests containing
+	// Query the manifest index for manifests containing
 	// the vulnerable indexrecords and create a set
-	// containing each unique manifest
+	// containing each unique manifest.
 	//
-	// since this loop opens a db conn each iteration the returned rows
+	// Since this loop opens a db conn each iteration the returned rows
 	// handle must be closed manually and not deferred else a buildup of db conns
-	// may occur until the method exits
+	// may occur until the method exits.
 	set := map[string]struct{}{}
 	out := []claircore.Digest{}
 	for _, record := range filteredRecords {
```

Comment:
As a boyscout task, what do you think about wrapping the code inside the for loop as a closure? Closing each iteration's row can happen as a defer then.